cmd/tikv-ctl/Cargo.toml

```diff
@@ -70,7 +70,7 @@ pd_client = { path = "../../components/pd_client", default-features = false }
 prometheus = { version = "0.13", features = ["nightly"] }
 protobuf = { version = "2.8", features = ["bytes"] }
 raft = { version = "0.7.0", default-features = false, features = ["protobuf-codec"] }
-raft-engine-ctl = { git = "https://github.com/tikv/raft-engine.git" }
+raft-engine-ctl = { git = "https://github.com/jiayang-zheng/raft-engine.git", rev = "9200bdae002a2fa09a56e0753eae211477f03f48" }
```

Comment:
Why do u introduce a local / private dependency path here?

Reply:
Work in process. I will propose a PR in raft-engine repo after test.